tests/common/helpers/ptf_tests_helper.py

```diff
@@ -113,16 +114,17 @@ def apply_dscp_cfg_teardown(duthost):
     reload_required = False
     for asic_id in duthost.get_frontend_asic_ids():
         swss = 'swss{}'.format(asic_id if asic_id is not None else '')
-        file_exists = duthost.shell("docker exec {} ls /usr/share/sonic/templates/ipinip.json.j2.tmp"
-                                    .format(swss))["rc"] == 0
-        if not file_exists:
+        try:
+            file_out = duthost.shell("docker exec {} ls /usr/share/sonic/templates/ipinip.json.j2.tmp".format(swss))
```

Comment:
Why do we need to change this part?

Reply:
ansible immediately returns a failure if the file doesn't exist so I am putting it in a try catch block to prevent an immediate failure